blobfuse/OAuthToken.cpp

```diff
@@ -23,13 +25,18 @@ void to_json(json &j, const OAuthToken &t) {
 }
 
 void from_json(const json &j, OAuthToken &t) {
+    // JUST PRINT OUT THE STRIng SO THAT WE KNOW THE CUSTOM ENDPOINT IS SENDING THE RIGHT JSON, useful for debugging OAuthtoken errors
+    std::string s = j.dump();   
+    syslog(LOG_WARNING, "Printing Json Token as string %s\n", s.c_str());

```

Comment:
Should this be on LOG_WARNING, shouldn't this be on LOG DEBUG?

Also just to verify, when we call this method, there's no chance of us printing out any secure customer information to the logs?

Reply:
It should really be LOG_INFO. But ... They mostly don't turn debug on and for troubleshooting with log this will be useful so I upgraded it to warning. I can change text to ay Just INFO: Not warning. With so many varieties in date this there will be lot more date troubleshooting happening. I was just writing a TSG for date troubleshooting which included checkign this WARNING but INFO